README.md

```diff
@@ -1,12 +1,41 @@
-# Git Repository Template
+# Internal Certification Challenge
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+A project to challenge ourselves with specific web stacks to achieve the Internal Certification. 🚀
+
+The requirement of this project is about creating a surveyor with different types of questions such as the list below here.
+
+#### Type of questions:
+* Listing
+* Yes/No
+* Rating
+* NPS
+* Text input
+* Textarea input
+* Select option
+* Multiple choice
+* Slider
+
+## Prerequisite
+* [Next.js](https://github.com/vercel/next.js)
```

Comment:
Maybe add the node version here too? 💭

Reply:
Added node into the list of prerequisites https://github.com/Lahphim/react-surveyor-challenge/pull/44/commits/7304117282ca348b8a3dcc5bcadd5629fb634ada